_translator-files/po/de/Installation-for-Windows.po

```diff
@@ -266,7 +263,7 @@ msgstr "Danach versuche *die Ein-/Ausgänge nochmal neu einzurichten*. Aktiviert
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:77
 msgid "Have a look at [this video](https://youtu.be/_GzOsitVgLI) by [trombonepizza](https://github.com/trombonepizza) which gives more detailed setup information on ASIO4ALL."
-msgstr "Wirf einen Blick auf [dieses Video](https://youtu.be/_GzOsitVgLI){: target=\"_blank\" rel=\"noopener noreferrer\"} von [trombonepizza](https://github.com/trombonepizza), das ausführlichere Informationen zur Einrichtung von ASIO4ALL gibt (English)."
+msgstr "Werfe einen Blick auf [dieses Video](https://youtu.be/_GzOsitVgLI) von [trombonepizza](https://github.com/trombonepizza), das detailliertere Informationen zur Einrichtung von ASIO4ALL enthält."
```

Comment:
Maybe we could also find a German ASIO4ALL video here (similar to IT or FR - at least that's what I remember)